src/NServiceBus.Gateway.Tests/APIApprovals.Approve.approved.txt

```diff
@@ -141,7 +142,7 @@ namespace NServiceBus
     public class GatewaySettings
     {
         public void AddReceiveChannel(string address, string type = "http", int maxConcurrency = 1, bool isDefault = False) { }
-        public void AddSite(string siteKey, string address, string type = "http", bool legacyMode = False) { }
+        public void AddSite(string siteKey, string address, string type = "http", bool legacyMode = False, bool usesReverseProxy = False) { }
```

Comment:
While this has a default value for the new parameter in the signature, does it constitute a breaking change in the semver world? Part of me says no (users don't need to change anything because of the default value), part of me says yes because the public API is actually changing. Right now I'm leaning towards calling this a breaking change. 

@Particular/nservicebus-maintainers  your thoughts?

Reply:
Optional parameters are baked into the call site at compile time, so this is definitely a binary breaking change.

To do this is a non-breaking way, it would need to be a new overload that adds the new parameter instead.